docs/src/app/components/pages/components/date-picker.jsx

```diff
@@ -90,21 +90,21 @@ export default class DatePickerPage extends React.Component {
           },
           {
             name: 'maxDate',
-            type: 'date object',
+            type: 'date',
```

Comment:
ah, that mirrors a propType. sure, in that case i guess there is some sense to it :)